boards/arm/adafruit_kb2040/doc/index.rst

```diff
@@ -0,0 +1,118 @@
+.. _adafruit_kb2040:
+
+Adafruit KB2040
+###############
+
+Overview
+********
+
+The Adafruit KB2040 is a small, low-cost, versatile board from
+Adafruit. It is equipped with an RP2040 SoC, an on-board RGB Neopixel,
+a USB connector, and an QT connector. The USB bootloader allows it
+to be flashed without any adapter, in a drag-and-drop manner.
+
+Hardware
+********
+- Dual core Arm Cortex-M0+ processor running up to 133MHz
+- 264KB on-chip SRAM
+- 2MB on-board QSPI flash with XIP capabilities
+- 26 GPIO pins
```

Comment:
Are all of them accessible on this board?

Reply:
No. Fixed.